src/components/colorPalette/index.tsx

```diff
@@ -331,19 +336,19 @@ class ColorPalette extends PureComponent<Props, State> {
   }
 
   renderScrollableContent() {
-    const {containerStyle, ...others} = this.props;
+    const {containerStyle, backgroundColor, ...others} = this.props;
     const {scrollable, contentWidth} = this.state;
 
     return (
       <ScrollBar
         ref={this.scrollBar}
-        style={[containerStyle, styles.scrollContainer]}
+        style={[containerStyle, styles.scrollContainer, {backgroundColor}]}
```

Comment:
Why not pass the backgroundColor in the containerStyle and just not override it? Both 'style' and 'gradientColor' can get the bg color from the same source `const bg = style?.backgroundColor || Colors.$backgroundDefault;`